std/algebra/emulated/sw_emulated/point.go

```diff
@@ -522,60 +522,160 @@ func (c *Curve[B, S]) scalarMulGLV(Q *AffinePoint[B], s *emulated.Element[S], op
 		one := c.baseApi.One()
 		Q = c.Select(selector, &AffinePoint[B]{X: *one, Y: *one}, Q)
 	}
-	sd, err := c.scalarApi.NewHint(decomposeScalarG1, 5, s, c.eigenvalue, frModulus)
+	sd, err := c.scalarApi.NewHint(decomposeScalarG1, 7, s, c.eigenvalue, frModulus)
 	if err != nil {
 		panic(fmt.Sprintf("compute GLV decomposition: %v", err))
 	}
-	s1, s2, s3, s4 := sd[0], sd[1], sd[3], sd[4]
+	s1, s2, s3, s4, s5, s6 := sd[0], sd[1], sd[3], sd[4], sd[5], sd[6]
 
 	c.scalarApi.AssertIsEqual(
-		c.scalarApi.Add(s1, c.scalarApi.Mul(s2, c.eigenvalue)),
+		c.scalarApi.Add(s5, c.scalarApi.Mul(s6, c.eigenvalue)),
 		c.scalarApi.Add(s, c.scalarApi.Mul(frModulus, sd[2])),
```

Comment:
Do we need to output `sd[2]` and check here using `sd[2]`? Wouldn't it be sufficient if we only check
```go
c.scalarApi.AssertIsEqual(
		c.scalarApi.Add(s5, c.scalarApi.Mul(s6, c.eigenvalue)),
        s
)
```
as AsssertIsEqual perform equality check modulo `frModulus`?

Reply:
you're right! that was an old copy-paste from 2-chains